### DISCUSSION TITLE
Erster Test / Feedback

### DISCUSSION THREAD

**Discussion by nwju848** [mbridak/not1mm#173] [Q&A]
Hallo zusammen,

ich habe die letzten Tage das Programm getestet und wollte nun einmal Feedback geben. evtl. hilft es Ihnen weiter und oder Sie können mir Helfen es zu laufen zu bringen.

1. Beim starten des Programmes kommt die Meldung "Unable to locate or open the VFO knob serial device". Cat Steuerung funktioniert aber soweit über Flrig. Frequenz im VFO Fenster wird in rot angezeigt.
2. Das DX Cluster funktioniert nur sporadisch und wechselt nicht das Band beim wechseln am Rig. Es steht bei mir immer auf dem 20m Band. VFO Indikator ist nicht zu sehen.
3. Ein neues call wird erst nach dem Neustart des Programmes in der QSO History angezeigt.
4. Im CheckPartial wird nichts angezeigt.

Danke für Ihre Arbeit.

Gruß Alex

**Comment by mbridak** [comment]
Hallo Alex,

danke für den Bericht. Ein paar Dinge, die ich am Anfang klären kann, sind die Anmerkungen zu der Meldung, dass der VFO-Knopf nicht gefunden werden konnte und die Frequenz rot sei. Beide beziehen sich auf ein Hardwareprojekt für einen VFO-Knopf, der bei der Fernsteuerung eines Radios verwendet werden soll. Weitere Informationen finden Sie unter [VFO KNOB](https://github.com/mbridak/not1mm/blob/master/usb_vfo_knob/vfo.md). Wenn Sie den Knopf nicht gebaut haben oder nicht vorhaben, ein Radio fernzusteuern, wäre es am besten, wenn Sie dieses Fenster nicht aktiv lassen.

Was die anderen Probleme betrifft, stimmt etwas mit den Multicast-UDP-Paketen nicht. Die Programme kommunizieren über Multicast-UDP-Pakete miteinander, um die Teile mitzuteilen, die sie zum Beispiel zum Aktualisieren des QSO-Protokollfensters, zum Ändern der angezeigten Bänder oder zum Ändern des Rufzeichens benötigen, und das Programm sollte seine Liste der passenden Rufzeichen aktualisieren.

Ich hoffe, das hilft.
Mike K6GTE

**Comment by mbridak** [comment]
Fast vergessen, was die sporadischen Cluster-Flecken angeht. Ich müsste sehen, welche Einstellungen Sie für den Cluster haben. Es kann sein, dass Sie sie auf ihren Standardeinstellungen belassen haben, was für Sie nicht gut funktionieren würde. Ich glaube, in den Standardeinstellungen steht so etwas wie „Set DX Filter SpotterCont=NA“. Aber so etwas wie „Set DX Filter (SKIMMER and SpotterCont=EU)“ würde für Sie wahrscheinlich besser funktionieren.

**Comment by nwju848** [comment]
Danke für die schnelle Antwort. Nach dem update 24.10.24 geht nun alles. Gruß Alex
